recipe/meta.yaml

```
@@ -76,19 +80,25 @@ outputs:
   - name: python-gmsh
     build:
       noarch: python
-      skip: true  # [build_platform != target_platform]
-    script: install_python.sh  # [not win]
-    script: install_python.bat  # [win]
+      skip: true  # [build_platform != target_platform or target_platform != "linux-64" or occt != "7.8.1"]
+    script: install_python_module.py
     requirements:
+      host:
+        - python {{ python_min }}
       run:
         # this should pin build number as well,
         # but we can't use exact=True while this is noarch
         - {{ pin_subpackage("gmsh", max_pin="x.x.x") }}
```

The simplest way to do this IMO would be to just depend on `gmsh` here more generically. During each build process, this will be satisfied by the local artefacts, but the `python-gmsh` builds for different occt versions will have exactly the same dependencies (and thus build string hashes), and so their uploads should actually coincide (i.e. the second one gets ignored if an artefact with exactly matching `package-version-buildstring` exists already).
```suggestion
      run:
        - gmsh {{ version }} *_{{ PKG_BUILDNUM }}
```